proprietary/design-tokens/src/component/utrecht/calendar.tokens.json

```diff
@@ -0,0 +1,177 @@
+          "font-weight": {
+            "value": ""
+          },
+          "hover": {
+            "background-color": {
+              "value": ""
+            },
+            "color": {
+              "value": "{utrecht.color.blue.40}"
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            }
+          },
+          "out-of-the-month": {
+            "color": {
+              "value": "{utrecht.color.grey.80}"
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "is-today": {
+            "color": {
+              "value": "{utrecht.color.blue.40}"
+            },
+            "font-weight": {
+              "value": "700"
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "emphasis": {
+            "color": {
+              "value": ""
+            },
+            "font-weight": {
+              "value": ""
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "selected": {
+            "color": {
+              "value": ""
+            },
+            "font-weight": {
+              "value": ""
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "disabled": {
+            "color": {
+              "value": ""
+            },
+            "font-weight": {
+              "value": ""
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          }
+        }
+      },
+      "navigation": {
+        "background-color": {
+          "value": "{utrecht.color.black}"
+        },
+        "color": {
+          "value": "{utrecht.color.white}"
+        },
+        "padding-block": {
+          "value": "20px"
+        },
+        "label": {
+          "min-width": {
+            "value": "200px"
+          },
+          "color": {
+            "value": "{utrecht.color.white}"
+          }
+        },
+        "buttons": {
+          "item-icon": {
```

Comment:
`item-icon` ➝ `icon`

Reply:
If we change it to `icon ` we will make the token different from the CSS class
Is that okay?